### PR TITLE
Point to new miniconda url and use wget to follow redirects

### LINKED ISSUE #103: Miniconda download location changed
The download page for miniconda changed from https://repo.continuum.io/miniconda/  to https://repo.anaconda.com/miniconda/

This breaks the miniconda installation in `before_install.sh`.

The following line will need to be updated in all cookiecutter-based repos: https://github.com/MolSSI/cookiecutter-cms/blob/1a67980182f41598b29acf00d26909b77fd29d83/%7B%7Bcookiecutter.repo_name%7D%7D/devtools/travis-ci/before_install.sh#L18

Two current options are:
* Changing the URL directly
* Changing from `curl -s` --> `wget -qO-`

### PR DESCRIPTION
**Pull request by j-wags**
Closes #103

### CODE REVIEW

File: {{cookiecutter.repo_name}}/devtools/travis-ci/before_install.sh [hunk 1]
```diff
@@ -15,8 +15,8 @@ else
     MINICONDA=Miniconda3-latest-Linux-x86_64.sh
 fi
 MINICONDA_HOME=$HOME/miniconda
-MINICONDA_MD5=$(curl -s https://repo.continuum.io/miniconda/ | grep -A3 $MINICONDA | sed -n '4p' | sed -n 's/ *<td>\(.*\)<\/td> */\1/p')
-wget -q https://repo.continuum.io/miniconda/$MINICONDA
+MINICONDA_MD5=$(curl -s https://repo.anaconda.com/miniconda/ | grep -A3 $MINICONDA | sed -n '4p' | sed -n 's/ *<td>\(.*\)<\/td> */\1/p')
```
Comment (loriab) [inline {{cookiecutter.repo_name}}/devtools/travis-ci/before_install.sh:18]: title says switching to `wget` but code doesn't?

Comment (j-wags) [reply {{cookiecutter.repo_name}}/devtools/travis-ci/before_install.sh]: Oops. Fixed in 2052a7c

File: {{cookiecutter.repo_name}}/devtools/travis-ci/before_install.sh [hunk 2]
```diff
@@ -15,8 +15,8 @@ else
     MINICONDA=Miniconda3-latest-Linux-x86_64.sh
 fi
 MINICONDA_HOME=$HOME/miniconda
-MINICONDA_MD5=$(curl -s https://repo.continuum.io/miniconda/ | grep -A3 $MINICONDA | sed -n '4p' | sed -n 's/ *<td>\(.*\)<\/td> */\1/p')
-wget -q https://repo.continuum.io/miniconda/$MINICONDA
+MINICONDA_MD5=$(wget -qO- https://repo.anaconda.com/miniconda/ | grep -A3 $MINICONDA | sed -n '4p' | sed -n 's/ *<td>\(.*\)<\/td> */\1/p')
```
Comment (dgasmith) [inline {{cookiecutter.repo_name}}/devtools/travis-ci/before_install.sh:18]: `wget` follows redirect by default which is good. `curl` can with the `-L` flag as well, but standardizing on `wget` is good.

**Comment by dgasmith** [comment]
Can you minor bump the minor version in the `cookiecutter.json`, will merge after.

**Comment by j-wags** [comment]
@dgasmith I've bumped the minor version in `cookiecutter.json` from `1.1` to `1.2`

**Pull request merged**